cpp/open3d/data/Downloader.h

```diff
@@ -35,28 +35,22 @@ namespace data {
 /// \param file_path Path to the file.
 std::string GetSHA256(const std::string& file_path);
 
-/// \brief Function to download the file from URL.
-/// \param url URL for the file to be downloaded.
-/// \param output_file_path Custom directory to download the file. If
-/// directory does not exists, it will be created. If empty string is
-/// passed, the default data-root will be used.
-/// \param output_file_name Name of the downloaded file. If empty string is
-/// passed, the default file name will be used, extracted from the url.
-/// \param always_download If `false`, it will skip download if the file is
-/// present in the given location with given file name and expected SHA256SUM.
-/// It will trigger download if these conditions are not met. If `true`, it will
-/// always trigger download and over-write the file if present. Default: `true`.
-/// \param sha256 SHA256SUM HASH value to verify the file after download. If
-/// empty string is passed, the verification will be skipped. If
-/// `always_download` is set to `false`, then it `SHA256` is a required
-/// parameter.
-/// \param print_progress Display progress bar for download.
-bool DownloadFromURL(const std::string& url,
-                     const std::string& output_file_path = "",
-                     const std::string& output_file_name = "",
-                     const bool always_download = true,
-                     const std::string& sha256 = "",
-                     const bool print_progress = false);
+/// \brief Download a file from URL.
+///
+/// \param url File URL.
```

Comment:
nit: do we want to mention somewhere, that the file name is extracted from the last part of the URL?

Reply:
Added.